packages/application-generic/src/services/bull-mq.service.ts

```diff
@@ -114,4 +117,50 @@ export class BullmqService {
         : {}),
     });
   }
+
+  public async gracefulShutdown(): Promise<void> {
+    Logger.log('Shutting the BullMQ service down', LOG_CONTEXT);
+
+    await this._queue.close();
```

Comment:
Might we want to check if `this._queue` exists in case BullMqService is only used to create a worker and not a queue in another service dependant on this.